spec/functional/honour_origin_cache_headers_spec.rb

```diff
@@ -1,9 +1,66 @@
 describe 'honour_origin_cache_headers' do
+
+  let(:a_date_in_the_future) { 'Thu, 01 Dec 2015 07:00:00 GMT' }
+  let(:a_date_in_the_future_plus_one) { 'Thu, 01 Dec 2015 07:01:00 GMT' }
```

Comment:
Perhaps, up to you. To be clear, you are proposing freezing at some fixed timestamp and testing against some other fixed timestamp?